plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -215,10 +216,15 @@ export class EventPipelineRunner {
             return this.runHeatmapPipelineSteps(event, kafkaAcks)
         }
 
-        const processedEvent = await this.runStep(pluginsProcessEventStep, [this, event], event.team_id)
+        const [postCookielessEvent] = await this.runStep(cookielessServerHashStep, [this, event], event.team_id)
+        if (postCookielessEvent == null) {
```

Comment:
Why is it `null` here if we're returning `undefined` instead? Is there a process step I'm missing?

Reply:
`== null` means equals null or undefined, I was being locally consistent with the rest of this file :)

Reply:
Oh, Javascript... Yeah, you're right :)